demo/app/models/example_form_builder/text_fields.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module ExampleFormBuilder
+  class TextFields < WizardSteps::Step
```

Comment:
Nitpick - just from a naming convention I feel like this should be singular `TextField` as the others are. Alternatively, maybe the others should be plural?